feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.ts

```diff
@@ -18,6 +18,7 @@ import { Configurator } from '../../core/model/configurator.model';
   providedIn: 'root',
 })
 export class ConfiguratorStorefrontUtilsService {
+  protected readonly CX_PREFIX = 'cx';
   protected readonly SEPARATOR = '--';
   protected readonly ADD_TO_CART_BUTTON_HEIGHT = 82;
```

Comment:
Please document, how is it connected to styling?

Reply:
Done!